modules/nvidia-gpu-admin-dashboard-using.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-using_{context}"]
+= Using the NVIDIA GPU administration dashboard
+
+After deploying the the OpenShift Console NVIDIA GPU plugin, log in to the OpenShift Container Platform web console using your login credentials to access the *Administrator* perspective.
+
+== Viewing the cluster GPU overview
+
+You can view the status of your cluster GPUs in the Overview page by selecting
+Overview in the Home section.
+
+The Overview page provides information about the cluster GPUs, including:
+
+* Details about the GPU providers
+* Status of the GPUs
+* Cluster utilization of the GPUs
+
+== Viewing the GPUs dashboard
+
+You can view the NVIDIA GPU administration dashboard by selecting GPUs
+in the Compute section of the OpenShift Console.
+
+Charts on the GPUs dashboard include:
+
+* GPU utilization
+* Memory utilization
+* Encoder utilization
+* Decoder utilization
+* Power consumption
+* GPU temperature
+* GPU clock speed
+* Memory clock speed
+
+Each chart type explains:
+
+* What the chart represents
+* Why the chart is important
+* Expected and typical results
+* How to potentially mitigate bad results
```

Comment:
I'd like to provide more information on this.

Reply:
I don't think we want to interpret metrics from the charts. This should be in a troubleshooting guide for the GPU Operator.

Reply:
@fabiendupont  This (lines 46-51) is a very high level overview. More detail would go into a troubleshooting guide. Do you want to leave this in or remove it?

Reply:
I agree with @fabiendupont that as a dashboard, it just visualizes metrics and potentially the state of the device but it doesn't interpret them in any way. I would be up for removing it.